packages/amplication-server/src/core/moduleAction/dto/EnumModuleActionType.ts

```diff
@@ -0,0 +1,19 @@
+import { registerEnumType } from "@nestjs/graphql";
+
+export enum EnumModuleActionType {
```

Comment:
also this

Reply:
@barshimi  Same like other comments above.